akka-actor/src/main/scala/akka/util/Helpers.scala

```diff
@@ -14,7 +14,11 @@ import java.util.Locale
 
 object Helpers {
 
-  val isWindows: Boolean = System.getProperty("os.name", "").toLowerCase(Locale.ROOT).indexOf("win") >= 0
+  implicit class LocaleUnawareString(s: String) {
```

Comment:
this is costly and magic, please don't add conversions to add methods :-)
just `Helpers.toRootLowerCase` would be good